R/dump.R

```diff
@@ -65,14 +65,35 @@ stone_dump <- function(con, touchstone, path, include_deps = FALSE) {
 
   #########################################################################
 
+  dump_scenario_type <- function() {
+
+    # Note that focal_coverage_set info is not dumped, since
```

Comment:
is this comment out of date/sync?

Reply:
It's still true, stoner doesn't do anything with coverage (yet), so doesn't export it. 

I think there are two paths forward on coverage... possibly both implementable...

1. Could export the numerical id for focal_coverage_set, so that a touchstone can be duplicated retaining the original's coverage sets... 

2.  The bigger, more grizzly chunk of work would be how to represent the coverage as CSVs and provide some stoner tools to manage them. I'm not sure whether it's possible to really make something nice, or whether fixing/updating coverage is just too ad hoc to write very good tools for...